src/gql/generated/gql.ts

```diff
@@ -14,7 +14,7 @@ import type { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-
  * Learn more about it here: https://the-guild.dev/graphql/codegen/plugins/presets/preset-client#reducing-bundle-size
  */
 type Documents = {
-  '\n  fragment AccountItem on Account {\n    nodeId\n    id\n    username\n  }\n': typeof types.AccountItemFragmentDoc
+  '\n  fragment AccountItem on Account {\n    nodeId\n    id\n    username\n    description\n  }\n': typeof types.AccountItemFragmentDoc
```

Comment:
This is functionality available on `beta` only. Please target branch beta and mark this PR as a breaking change by naming it `feat(account)!: restyle profile page` (exclamation mark after scope)

Reply:
Btw you can see that this is indeed breaking, when you start frontend only development and click on an event in the discover list

Reply:
I merged in `beta` for you, so you don't need to make any code changes, just set the PR's target branch to beta and add the exclamation mark

Reply:
I'm merging `beta` into `main`, so the above should not be necessary anymore in a few minutes.